src/middlewared/middlewared/plugins/vmware.py

```diff
@@ -191,7 +191,8 @@ def get_datastores(self, data):
                         'remote_hostnames': host_mount_info.volume.remoteHostNames,
                         'username': host_mount_info.volume.userName,
                     }
-                elif host_mount_info.volume.type == 'OTHER':
+                elif host_mount_info.volume.type == 'OTHER' or host_mount_info.volume.type == 'VFFS':
```

Comment:
Can we use `host_mount_info.volume.type in ('OTHER', 'VFFS')` instead?

It more clear and avoid two extra attribute access.

Reply:
Sure, no problem.